src/main/java/com/mule/einstein/internal/models/ParamsModelDetails.java

```diff
@@ -15,30 +16,30 @@ public class ParamsModelDetails {
   @Expression(ExpressionSupport.SUPPORTED)
   @OfValues(ModelApiNameProvider.class)
   @Optional(defaultValue = OPENAI_GPT_3_5_TURBO)
```

Comment:
Same here as well, should we replace with API names instead?

Reply:
As mentioned in previous comment